src/elements/checkbox.scss

```diff
@@ -79,12 +83,16 @@ $padding-label-toggle: 63px;
             .ui.icon-check { left: auto; right: 6px; }
         }
     }
-    &-disabled {
-        label {
-            color: color(colorStatic);
+    &-disabled .input {
+        + label, &:checked + label {
             cursor: auto;
-            &::before { background: color(backgroundColorDisable); }
-            .ui.icon-check { color: color(backgroundColorStatic); }
+            &::before {
+                background-color: color(backgroundColorDisable);
+                border: 1px solid color(borderColor);
+            }
+        }
+        &:checked + label .ui.icon-check {
+            opacity: 1;
```

Comment:
Do these changes by chance address that long-standing issue where disabled checkboxes would always appear to be checked?  If not, no biggie.  It looks though like it was perhaps getting into that area.  If so, I'm excited, and many thanks!

Reply:
Yes